docs/excel/custom-functions-json.md

```diff
@@ -122,6 +122,7 @@ The `options` object enables you to customize some aspects of how and when Excel
 |:-----|:-----|:-----|:-----|
 |  `cancelable`  |  boolean  |  No<br/><br/>Default value is `false`.  |  If `true`, Excel calls the `onCanceled` handler whenever the user takes an action that has the effect of canceling the function; for example, manually triggering recalculation or editing a cell that is referenced by the function. If you use this option, Excel will call the JavaScript function with an additional `caller` parameter. (Do ***not*** register this parameter in the `parameters` property). In the body of the function, a handler must be assigned to the `caller.onCanceled` member. For more information, see [Canceling a function](custom-functions-overview.md#canceling-a-function). |
 |  `stream`  |  boolean  |  No<br/><br/>Default value is `false`.  |  If `true`, the function can output repeatedly to the cell even when invoked only once. This option is useful for rapidly-changing data sources, such as a stock price. If you use this option, Excel will call the JavaScript function with an additional `caller` parameter. (Do ***not*** register this parameter in the `parameters` property). The function should have no `return` statement. Instead, the result value is passed as the argument of the `caller.setResult` callback method. For more information, see [Streaming functions](custom-functions-overview.md#streaming-functions). |
+|  `volatile`  | boolean | No <br/><br/>Default value is `false`. | If `true`, the function will recalculate each time Excel recalculates, instead of only when the formula's dependent values have changed. Functions cannot be both streaming and volatile. If marked with both options as `true`, the function will default to streaming `true` and volatile `false`. |
```

Comment:
- Suggest rephrasing the second sentence to ->  A function cannot be both streaming and volatile. 

- Suggest rephrasing the final sentence to ->  If the `stream` and `volatile` properties are both set to `true`, the `volatile` option will be ignored.